.github/actions/InstallPackages/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Install packages"
+description: "Installs packages that the hls-test-suite depends on."
+
+inputs:
+  llvm-version:
+    description: "LLVM/ version that is installed"
+    default: "17"
+    required: false
```

Comment:
Remove the `default` and make the required `true`, so it is necessary to hand it in from the site where it is invoked.

Reply:
Removed default and changed to required.